beacon_node/eth2-libp2p/Cargo.toml

```diff
@@ -7,15 +7,17 @@ edition = "2018"
 [dependencies]
 beacon_chain =  { path = "../beacon_chain" }
 clap = "2.32.0"
-# SigP repository until PR is merged
-libp2p =  { git = "https://github.com/SigP/rust-libp2p", rev = "b3c32d9a821ae6cc89079499cc6e8a6bab0bffc3" }
+# SigP repository 
+ libp2p =  { git = "https://github.com/SigP/rust-libp2p", rev = "8ff9f2001de0aea1175c1442f22bfbf382dc0e8b" }
```

Comment:
Weird leading space?

Reply:
Yeah I saw that also....it's already fixed locally, it will get fixed in the wash.